Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7
+
+RUN groupadd --gid 10001 app && \
+    useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
+
+USER app
+WORKDIR /app
+
+COPY . /app
+
+RUN python -m venv /app
+RUN ./bin/pip install -r requirements/base.txt
```

Comment:
I think you need to install [configloader](https://github.com/mozilla-releng/beetmoverscript/blob/d5cb318d474dac7729af6b3c383d61ea071c22b6/Dockerfile#L14) in order to preprocess the configs in `init.sh`